ui/apps/platform/src/Containers/Policies/Modal/ImportPolicyJSONModalError.tsx

```diff
@@ -66,7 +66,9 @@ function ImportPolicyJSONError({
         >
             <>
                 <ModalBoxBody>
-                    Address the errors below to continue importing policies
+                    {duplicateErrorsOnly
+                        ? 'Address the errors below to continue importing policies'
+                        : 'Correct the file to address the errors listed below, and then import again.'}
```

Comment:
Similar to comment above, is there a more precise verb than **address** which is also noun?

Docs team can usually suggest alternatives.

Reply:
@kcarmichael08 @gaurav-nelson 

(similar to request in my other comment)

Could one of you suggest something clearer for this sub-heading when there are multiple policy import errors?

Reply:
Correct the errors that are listed, and then try to import again.

(Generally in docs we try to avoid using directional terms like "below" "above" etc. because sometimes GUI elements move, sometimes people use screenreaders and don't know where the elements are, etc. I don't know if this applies to GUI text also, but it's a docs accessibility thing.)